include/podio/UserDataCollection.h

```diff
@@ -117,6 +117,10 @@ class UserDataCollection : public CollectionBase {
             }};
   }
 
+  podio::CollectionReadBuffers createSchemaEvolvableBuffers(__attribute__((unused)) int readSchemaVersion, __attribute__((unused)) podio::Backend backend) /*const*/ final {
```

Comment:
```suggestion
  podio::CollectionReadBuffers createSchemaEvolvableBuffers(int, podio::Backend) /*const*/ final {
```
Same effect as the attribute. In any case if we go the attribute route, we should use [`[[maybe_unused]]`](https://en.cppreference.com/w/cpp/language/attributes/maybe_unused)